css/modern.css

```diff
@@ -316,6 +316,22 @@ footer button,
     color: var(--contrast-text) !important;
 }
 
+
+.action-links-wrapper-expanded .action-links-unfold .action-links-unfold-text {
+    background-color:var(--text) !important;
```

Comment:
I'm not entirely certain what section this is for, but why is the background color a text color?